code/ai/aigoals.cpp

```diff
@@ -2547,7 +2547,7 @@ void ai_update_goal_references(ai_goal *goals, int type, const char *old_name, c
 				break;
 
 			default:
-				Warning(LOCATION, "unhandled FRED reference type %d in ai_update_goal_references", type);
+				Warning(LOCATION, "unhandled FRED reference type %d in ai_update_goal_references", (int)type);
```

Comment:
_scoped_ enum casts should really be done using static_cast only

Reply:
ok